src/propIs.js

```diff
@@ -0,0 +1,28 @@
+var _curry3 = require('./internal/_curry3');
+var is = require('./is');
+var propSatisfies = require('./propSatisfies');
+
+
+/**
+ * Returns `true` if the specified object property is of the given type;
+ * `false` otherwise.
+ *
+ * @func
+ * @memberOf R
+ * @category Type
+ * @sig Type -> String -> Object -> Boolean
+ * @param {Function} type
+ * @param {String} name
+ * @param {*} obj
+ * @return {Boolean}
+ * @see R.is
+ * @see R.propSatisfies
+ * @example
+ *
+ *      R.propIs('x', Number, {x: 1, y: 2});  //=> true
+ *      R.propIs('x', Number, {x: 'foo'});    //=> false
+ *      R.propIs('x', Number, {});            //=> false
+ */
+module.exports = _curry3(function propIs(type, name, obj) {
+  return propSatisfies(is(type), name, obj);
+});
```

Comment:
I was looking for that gist! I'd forgotten that it's a gist rather than a pull request, so couldn't find it.

> How do we know how many parameters each intermediate function should consume?

Each function would consume as many of the original arguments as are necessary to satisfy it. I see no ambiguity.

Composing unary functions:

              +-----+             +-----+                +-----+
    --- a --->|  f  |--- f(a) --->|  g  |--- g(f(a)) --->|  h  |--- h(g(f(a))) --->
              +-----+             +-----+                +-----+

Composing binary functions:

              +-----+
    --- a --->|     |
              |  f  |                +-----+
    --- b --->|     |--- f(a, b) --->|     |
              +-----+                |  g  |                      +-----+
    --- c -------------------------->|     |--- g(f(a, b), c) --->|     |
                                     +-----+                      |  h  |
    --- d ------------------------------------------------------->|     |--- h(g(f(a, b), c), d) --->
                                                                  +-----+

Composing a unary function, a binary function, and a ternary function:

              +-----+             +-----+
    --- a --->|  f  |--- f(a) --->|     |
              +-----+             |  g  |                   +-----+
    --- b ----------------------->|     |--- g(f(a), b) --->|     |
                                  +-----+                   |     |
    --- c ------------------------------------------------->|  h  |
                                                            |     |
    --- d ------------------------------------------------->|     |--- h(g(f(a), b), c, d) --->
                                                            +-----+